src/inspector/models/abstractinspectormodel.cpp

```diff
@@ -89,6 +89,8 @@ static const QMap<mu::engraving::ElementType, InspectorModelType> NOTATION_ELEME
     { mu::engraving::ElementType::TUPLET, InspectorModelType::TYPE_TUPLET },
     { mu::engraving::ElementType::TEXTLINE, InspectorModelType::TYPE_TEXT_LINE },
     { mu::engraving::ElementType::TEXTLINE_SEGMENT, InspectorModelType::TYPE_TEXT_LINE },
+    { mu::engraving::ElementType::TEMPO_RANGED_CHANGE, InspectorModelType::TYPE_TEMPO_RANGED_CHANGE },
+    { mu::engraving::ElementType::TEMPO_RANGED_CHANGE_SEGMENT, InspectorModelType::TYPE_TEMPO_RANGED_CHANGE },
```

Comment:
In line with https://github.com/musescore/MuseScore/issues/11251#issuecomment-1105174396 ...

Reply:
TEMPO_RANGED_CHANGE is not grammatically correct and should be fixed.  But also, while it might be the same error made as elsewhere in the code, it's a *different* error than the one made in the status line, where it says "Tempo ranged changed".  There is no past tense at play here, the letter "d" should not appear at all in this context.

But yeah, if it's not too late, I definitely vote for something more like "gradual tempo change".

Reply:
I might totally misinterpret it, but I read `TEMPO_RANGED_CHANGE` as a "ranged tempo change", i.e. a "tempo change" that takes place over a "range" of measures.

Reply:
Hmm, when you say it like thatmy brain can finally parse it - “tempo: ranged change”, like “recording: spoken word”.   But that’s still extremely awkward and just looks like a typo.  @bkunda, any better suggestion?

Reply:
I agree with @cbjeukendrup's parsing above ("ranged" as an adjective) and would also find `GRADUAL_TEMPO_CHANGE` much easier to make sense of.
That being said, naming elements like this in the code kind of lies beyond my area of expertise, so I would be happy to accept @cbjeukendrup's suggestion here.

Reply:
I'd propose to just merge this, and then I'll do a dedicated PR to rename all related things to GradualTempoChange etc.

Reply:
@cbjeukendrup thanks for the idea! I renamed TempoRangedChange globally

Reply:
That's great!